jvcl/run/JvWndProcHook.pas

```diff
@@ -10,22 +10,17 @@
 
 The Original Code is: JvWndProcHook.PAS, released on 2002-11-01.
 
-The Initial Developer of the Original Code is Peter Th�rnqvist [peter3 at sourceforge dot net]
-Portions created by Peter Th�rnqvist are Copyright (C) 2002 Peter Th�rnqvist.
+The Initial Developer of the Original Code is Peter Thörnqvist [peter3 at sourceforge dot net]
```

Comment:
This should not have changed, so it seems the encoding of the file itself has changed, which will break compatibility with some supported versions.